Users.md

```diff
@@ -26,6 +26,7 @@ Currently the MASVS and MSTG are used by at least the following companies:
 - [Stingray Technologies](https://stingray.appsec.global/  "Stingray Technologies")
 - [STM Solutions](https://stmsolutions.pl/ "STM Solutions")
 - [Toreon](https://www.toreon.com/ "Toreon")
+- [Trustwave](https://www.trustwave.com/ "Trustwave")
```

Comment:
For such an addition we need a separate PR. Are you working for Trustwave? We'd need an official request from the company, telling why they would like to be listed, how they use our material, etc. Could you please ask them to do that? In one of our internal meetings we'll take a look at their request and if everything looks well we can add them to the list.

They can gladly send the request to both Sven.Schleier@owasp.org and Carlos.Holguera@owasp.org

```suggestion
```